### PR TITLE
Add sal-sciplat-lab

### PR DESCRIPTION
**Pull request by athornton**
I've made the JL build a staged step with another stage; sal-sciplat-lab builds on top of sciplat-lab.

### CODE REVIEW

File: pipelines/release/nightly_release.groovy [hunk 1]
```diff
@@ -143,51 +143,99 @@ notify.wrap {
...
-            // BASE_IMAGE is the registry repo name *only* without a tag
-            string(
-              name: 'BASE_IMAGE',
-              value: stackResults.docker_registry.repo,
-            ),
-          ],
-          wait: false,
-        )
-      } // retry
+      stages {
```
Comment (jhoblitt) [inline pipelines/release/nightly_release.groovy]: `stages` is a "declarative" pipeline construct which I don't think will work under "scripted" piepline.

File: pipelines/release/nightly_release.groovy [hunk 2]
```diff
@@ -143,51 +143,99 @@ notify.wrap {
...
+                  name: 'IMAGE_NAME',
+                  value: scipipe.release.step.build_sciplatlab.image_name,
+                ),
+                // BASE_IMAGE is the registry repo name *only* without a tag
+                string(
+                  name: 'BASE_IMAGE',
+                  value: stackResults.docker_registry.repo,
+                ),
+              ],
+              wait: false,
```
Comment (jhoblitt) [inline pipelines/release/nightly_release.groovy]: `sqre/infra/build-sciplatlab` is triggered here but the pipeline won't wait for it to finish before triggering `sqre/infra/build-salsciplatlab`

File: pipelines/sqre/infra/build_salsciplatlab.groovy [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  Integer timelimit  = params.TIMEOUT
+
+  def run = {
+    stage('checkout') {
+      def branch = 'prod'
+      if (jlBleed) {
+        branch = 'master'
+      }
+      git([
+        url: 'https://github.com/lsst-sqre/sal-sciplat-lab',
```
Comment (jhoblitt) [inline pipelines/sqre/infra/build_salsciplatlab.groovy]: Is this job identical to the original sciplatlab accept for the source gitr repo?

**Comment by athornton** [comment]
Closing; reworked as https://github.com/lsst-dm/jenkins-dm-jobs/pull/678

**Pull request closed without merging**